src/test/scala/grading/Lab2Tests.scala

```diff
@@ -440,21 +504,29 @@ class Lab2Grader extends JUnitSuite {
       val tests = List[CPUTestCase](
         CPUTestCase("fibonacci",
                     Map("single-cycle" -> 300, "five-cycle" -> 6, "pipelined" -> 6),
+                    Map(), 
+                    Map(),
                     Map(6->11),
                     Map(6->11,5->89),
                     Map(), Map()),
         CPUTestCase("naturalsum",
                     Map("single-cycle" -> 200, "five-cycle" -> 6, "pipelined" -> 6),
+                    Map(), 
+                    Map(),
                     Map(),
                     Map(5->55),
                     Map(), Map()),
         CPUTestCase("multiplier",
-                Map("single-cycle" -> 1000, "five-cycle" -> 6, "pipelined" -> 6),
-              Map(5->23,6->20),
+                    Map("single-cycle" -> 1000, "five-cycle" -> 6, "pipelined" -> 6),
+                    Map(), 
+                    Map(),
+                    Map(5->23,6->20),
                     Map(5->23*20),
                     Map(), Map()),
         CPUTestCase("divider",
                     Map("single-cycle" -> 1000, "five-cycle" -> 6, "pipelined" -> 6),
+                    Map(), 
+                    Map(),
```

Comment:
I notice that you never use this for checking or initializing. Is it necessary?

Reply:
I put it there as a place holder to initialize csr registers, but since I couldn't poke or peek because of the register names being optimized out, I left it there for the future when csr testing is resolved. Should I remove it?

Reply:
Yeah... I know it was a pain to add... but since you're not using it now, it really should wait to be added until it's needed.

Reply:
No problem. I'll go ahead and remove it.